integration/gifdecoder/src/test/java/com/bumptech/glide/integration/gifdecoder/ByteBufferGifDecoderTest.java

```diff
@@ -17,14 +14,15 @@
 import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
 import com.bumptech.glide.load.engine.bitmap_recycle.LruArrayPool;
 import com.bumptech.glide.load.resource.bitmap.DefaultImageHeaderParser;
-import com.bumptech.glide.tests.GlideShadowLooper;
+import com.bumptech.glide.testlib.GlideShadowLooper;
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.util.ArrayList;
 import java.util.List;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.Matchers;
```

Comment:
I think `Assert.*`, `Mockito.*`, `Matchers.*`, `ArgumentMatchers.*` etc should be always static imported.